lib/schedule/data.ex

```diff
@@ -89,7 +89,6 @@ defmodule Schedule.Data do
 
   @spec stop(t(), Stop.id()) :: Stop.t() | nil
   def stop(%__MODULE__{stops: stops}, stop_id), do: stops[stop_id]
-
```

Comment:
**nitpick:** keeping the blank line between different functions is probably more readable.